LayoutTests/svg/animations/animate-viewport-overflow-2-expected.txt

```diff
@@ -1,10 +1 @@
-CONSOLE MESSAGE: ReferenceError: Can't find variable: runRepaintAndPixelTest
-layer at (0,0) size 800x600
-  RenderView at (0,0) size 800x600
-layer at (0,0) size 800x116
-  RenderBlock {HTML} at (0,0) size 800x116
-    RenderBody {BODY} at (8,8) size 784x100
-      RenderBlock {DIV} at (0,0) size 100x100
-        RenderSVGRoot {svg} at (8,8) size 100x300
-          RenderSVGRect {rect} at (8,8) size 100x300 [fill={[type=SOLID] [color=#0000FF]}] [x=0.00] [y=200.00] [width=100.00] [height=100.00]
-        RenderText {#text} at (0,0) size 0x0
+
```

Comment:
why don't we see any repaints here? Animation should be "advancing" to end.

Reply:
I just tried to fix `console` warning but haven't looked into why it is not generating repaint rects but I have seen it in few other SVG animations repaint tests as well. Might need to look more deeply.